conda/cli/install.py

```diff
@@ -234,44 +234,6 @@ def install(args, parser, command='install'):
     ospecs = list(specs)
     add_defaults_to_specs(r, linked, specs, update=isupdate)
 
-    # Don't update packages that are already up-to-date
```

Comment:
Should we reserve this change for 4.2.x or stick it in 4.1.x?  Is it actually causing a bug in 4.1.x or just inefficiency?